common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/Controller.java

```diff
@@ -84,15 +84,15 @@
      * @return a java representation of the JSON object hierarchy,
      *     or a JSONObject representing {} if parsing fails
      */
-    public JSONValue parseJSON(String jsonStr) {
+    public static JSONValue parseJSON(String jsonStr) {
         try {
             return JSONParser.parseStrict(jsonStr);
-        } catch (Throwable t) {
+        } catch (Exception e) {
             // only shows up in eclipse dev mode
-            t.printStackTrace();
+            e.printStackTrace();
```

Comment:
use a logger rather than to print the trace

Reply:
@sophiesongge As I understood, the person who wrote this code at first didn't want to display the messages in the Portals Log window but only in eclipse so I left it as is.